pkg/apis/psmdb/v1/perconaservermongodbrestore_types.go

```diff
@@ -68,9 +72,60 @@ func (r *PerconaServerMongoDBRestore) CheckFields() error {
 	if len(r.Spec.ClusterName) == 0 {
 		return fmt.Errorf("spec clusterName field is empty")
 	}
-	if len(r.Spec.BackupName) == 0 && (len(r.Spec.StorageName) == 0 || len(r.Spec.Destination) == 0) {
-		return fmt.Errorf("fields backupName or storageName and destination is empty")
+
+	if r.Spec.PITR == nil {
```

Comment:
looks like we removed StorageName, Destination check for PITR here, but added very similar check for PITR in `pkg/controller/perconaservermongodbrestore/perconaservermongodbrestore_controller.go` lines 186-189 
please lets keep both checks in the same place

Reply:
moved it in cef1f56e2d33a37afb8895c93920bd919763c6cb